bitcoin/psbt.c

```diff
@@ -10,6 +10,8 @@
 #include <common/amount.h>
 #include <common/type_to_string.h>
 #include <common/utils.h>
+/* FIXME: this is ugly! */
+#include <external/libwally-core/src/ccan/ccan/base64/base64.h>
```

Comment:
Any reason to not just, you know --- include `ccan/base64` in our `ccan/` dir?

Reply:
(Had to think back why I did it this way!)

I think because this is an ugly hack anyway.  We *really* don't want to be doing this.  Let me see if I can clean up further...

Reply:
Fixed,